src/Components/Activities/Modals/Delete/deleteModal.module.css

```diff
@@ -0,0 +1,68 @@
+.modal {
+  position: absolute;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(0, 0, 0);
```

Comment:
To maintain consistency in the code, you can use one color format in the styles.